service/src/main/java/bio/terra/profile/service/profile/ProfileService.java

```diff
@@ -83,6 +84,10 @@ public void deleteProfile(UUID id, AuthenticatedUserRequest user) {
             .userRequest(user)
             .addParameter(ProfileMapKeys.PROFILE_ID, id)
             .addParameter(JobMapKeys.CLOUD_PLATFORM.getKeyName(), platform.name());
+    if (CloudPlatform.AZURE.equals(billingProfile.cloudPlatform())) {
```

Comment:
It seemed silly to add all the individual app coordinates separately.
It also seems (a little) silly to keep passing the id separately, and then to only add the profile when we need the app coordinates.
I also didn't want to rely on retrieving the profile from the DAO later on, since we're deleting it.

Should I just get rid of the profile id param and just use the profile? I think that would be cleaner.

Reply:
I don't think this code needs to be specific to Azure.

I think it makes sense to remove the profile_id parameter and just always add the profile itself.